src/common/index.ts

```diff
@@ -2,13 +2,34 @@ import * as constants from './constants'
 import * as errors from './errors'
 import * as validate from './validate'
 import * as serverInfo from './serverinfo'
+import {xAddressToClassicAddress} from 'ripple-address-codec'
+
+export function ensureClassicAddress(account: string): string {
+  // For rippled requests that use an account: always use a classic address.
+  // Except for special cases, X-addresses used for requests
+  // must not have an embedded tag. In other words,
+  // `tag` should be `false`.
+  try {
+    const {
+      classicAddress,
+      tag
+    } = xAddressToClassicAddress(account)
+    if (tag !== false) {
+      throw new Error('This command does not support the use of a tag. Use an address without a tag.')
```

Comment:
one tricky thing: this will throw inside of the try block, meaning it will be caught below and then immediately swallowed. If that's intentional, I'd say just `return account;` here.

Reply:
+1 this will be much simpler to spot these errors using a traditional if / else statement rather than trying to enumerate all cases. 

You are also assuming that anything that xAddressToClassicAddress threw was an error indicating that it was not a classic address, which while true today, isn't guaranteed to hold true in this library as it's code from a dependency.